src/openrct2-ui/windows/Map.cpp

```diff
@@ -589,7 +589,7 @@ static void window_map_scrollmousedown(rct_window* w, int32_t scrollIndex, const
         window_scroll_to_location(mainWindow, { mapCoords, mapZ });
     }
 
-    if (land_tool_is_active())
+    if (scenery_tool_is_active())
```

Comment:
Wrong function

Reply:
Argh thought i caught all of these

Reply:
I'm pretty sure this is the last one. I went over this diff carefully, with the help of regex to find this only exception.